aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -624,6 +575,83 @@ HTTP/2 is enabled by default. Connections fall back to HTTP/1.1 if an HTTP/2 con
 
 For an ASP.NET Core app that targets the .NET Framework, OPTIONS requests aren't passed to the app by default in IIS. To learn how to configure the app's IIS handlers in *web.config* to pass OPTIONS requests, see [Enable cross-origin requests in ASP.NET Web API 2: How CORS Works](/aspnet/web-api/overview/security/enabling-cross-origin-requests-in-web-api#how-cors-works).
 
+::: moniker range=">= aspnetcore-2.2"
+
+## Application Initialization Module and Idle Timeout
+
+When hosted in IIS by the ASP.NET Core Module version 2:
+
+* [Application Initialization Module](#application-initialization-module) &ndash; App's hosted [in-process](xref:fundamentals/servers/index#in-process-hosting-model) or [out-of-process](xref:fundamentals/servers/index#out-of-process-hosting-model), can be configured to start automatically on a worker process restart or server restart.
+* [Idle Timeout](#idle-timeout) &ndash; App's hosted [in-process](xref:fundamentals/servers/index#in-process-hosting-model) can be configured not to timeout during periods of inactivity.
+
+### Application Initialization Module
+
+*Applies to apps hosted in-process and out-of-process.*
+
+[IIS Application Initialization](/iis/get-started/whats-new-in-iis-8/iis-80-application-initialization) is an IIS feature that sends an HTTP request to the app when the app pool starts or is recycled. The request triggers the app to start. By default, IIS issues a request to the app's root URL (`/`) to initialize the app (see the [additional resources](#application-initialization-module-and-idle-timeout-additional-resources) for details on further configuration).
+
+Confirm that the IIS Application Initialization role feature in enabled:
+
+On Windows 7 or later desktop systems when using IIS locally:
+
+1. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen).
+1. Open **Internet Information Services** > **World Wide Web Services** > **Application Development Features**.
+1. Select the check box for **Application Initialization**.
+
+On Windows Server 2008 R2 or later:
+
+1. Open the **Add Roles and Features Wizard**.
+1. When you reach the **Select role services** panel, open the **Application Development** node.
```

Comment:
On the **Select role services** panel, open the **Application Development** node.